addon/components/-ember-popper-targeted-legacy.js

```diff
@@ -0,0 +1,94 @@
+import EmberPopperBase from './ember-popper-base';
+import { GTE_EMBER_1_13 } from 'ember-compatibility-helpers';
+import { addObserver, removeObserver } from '@ember/object/observers';
+import { argument } from '@ember-decorators/argument';
+import { computed } from 'ember-decorators/object';
+import { tagName } from 'ember-decorators/component';
+import { type } from '@ember-decorators/argument/type';
+
+@tagName('div')
+export default class EmberPopper extends EmberPopperBase {
```

Comment:
Its seems like a lot of this functionality is duplicated from `-ember-popper-legacy`, any chance we can extend instead of duplicating the logic?

Reply:
re extend: for sure, good catch.

re consistency: fixed both ember-popper-legacy and ember-popper-targeted-legacy to reflect the fact that they are legacy components.

Reply:
Turns out we can't actually add `Legacy` into either name since these files are moved around during the build phase. Easier to keep them as `EmberPopper` and `EmberPopperTargeted` to keep IDE linters happy.